### PR TITLE
Bug/110 fix stretched page

### LINKED ISSUE #110: [BUG]App UI stretches beyond viewport when user is on mobile device - laredotornado
Steps to reproduce

1. Enter the Heroku app on mobile
2. Observe any page in the app

Expected result: App can fit the screen size of the mobile device and the user can see the entire UI.

Actual result:  the UI stretches beyond the current viewport of the mobile device.

### PR DESCRIPTION
**Pull request by DalerLeo**
closes #110 

While opened on mobile devices the home page is going out of viewport because the logo had the fixed height. I just fixed it.
And The Drawer Icon was overflowing due to absolute positioning, fixed it too.
 
**Before**
image

**Overflowing Drawing Icon**
image

**After**
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,7 +18,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
-
+.idea
```
Comment (jwu910) [inline .gitignore:21]: This is an environment specific hidden file for macOS. This shouldn't be committed. You can ignore this in your global `.gitignore`

Comment (DalerLeo) [reply .gitignore]: Ok, I revolved issue, thanks for noting it

**Comment by DalerLeo** [comment]
Not relevant any more

**Comment by laredotornado** [comment]
Hi, I can pick this up.

**Comment by mat10tng** [comment]
I think this is done

**Pull request closed without merging**